package.json

```diff
@@ -40,10 +40,13 @@
     }
   },
   "dependencies": {
+    "@reach/dialog": "^0.7.0",
```

Comment:
https://reacttraining.com/reach-ui/dialog/ 

Modals have a lot of accessibility features we need that I didn't think we needed to spend dev time creating so I found this lib which focuses on compatibility/accessibility and isn't bloated.

Reply:
cc @bwalker8038

Reply:
@tkarssli – I'm currently working on a PR to add modals into the EDS that will be up this week.

Reply:
Removed these libs for now